api.ts

```diff
@@ -102,11 +102,11 @@ export class StatusCodeError extends Error {
    */
   statusCode: number;
   /**
-   * The raw body of the HTTP response.
+   * The body of the HTTP response, parsed as a JavaScript object if the contents were JSON data.
```

Comment:
I imagine we also do the XML parsing here? And the binary -> Buffer? If so it might be best to just say that we parse it. Ideally we'd link to the fetcher guide that talks about response parsing, but I think at the moment we don't have a good way to link from TypeDoc to mkdocs, given the variable leading path.

Reply:
Good point, done.